quinn-h3/src/qpack/encoder.rs

```diff
@@ -35,17 +35,17 @@ pub enum Error {
     UnknownPrefix,
 }
 
-pub fn encode<W: BufMut>(
+pub fn encode<W: BufMut, T: Iterator<Item = HeaderField>>(
```

Comment:
Maybe `IntoIterator` makes more sense here?

Reply:
Fixed in d47c107